src/ucs/algorithm/levenshtein_dist.c

```diff
@@ -13,36 +13,37 @@
 #include <ucs/sys/sys.h>
 #include <string.h>
 
-#define MAX_ENV_VAR_LENGTH 30
+#define MAX_UCX_ENV_VAR_LENGTH 30
 
-int ucs_levenshtein(const char *s1, const char *s2)
+int ucs_levenshtein(const char *str1, const char *str2)

```

Comment:
WDYT of completely removing "levenshtein" from every identifier (func, file name, etc)?
That's actually an "internal implementation detail", we can simply call it "string distance" or something like that.

Reply:
If I understand that correctly, "Levenstein" refers not to the way of implementation, but to the name of certain metric. But from my point of view, using this second name as a part of function name may be unclear for the people who don't know about that metric. With accordance to Wikipedia it can be also referred to as "edit distance" so I will propose to use this term as more informative.

Reply:
I think in this case "string distance" is more clear to someone who is not familiar with the concept of "edit distance".
also, if we will implement another "distance metric" in the future (Q-Grams for example), then "edit distance" will not describe it properly.